packages/components/src/components/lineage/LineageSummary.tsx

```diff
@@ -39,7 +39,7 @@ class LineageSummaryImpl extends PureComponent<InjectedLineage & LineageSummaryO
                 title={
                     groupName +
                     ' ' +
-                    (suffixes.has(groupName.toLowerCase()) ? suffixes.get(groupName.toLowerCase()) : defaultTitleSuffix)
+                    (suffixes.has(nodesByType[groupName].queryName) ? suffixes.get(nodesByType[groupName].queryName) : defaultTitleSuffix)
```

Comment:
any need for the toLowerCase() check anymore?

Reply:
No, I don't think so.  The toLowerCase was added because previously we were comparing against the queryTitle, which is a modification of the queryName.  That was still not enough to handle all cases, though, so this change makes queryName available explicitly so we can compare to it directly.